### DISCUSSION TITLE
AX3600多次编译错误

### DISCUSSION THREAD

**Discussion by audyfan** [Boos4721/openwrt#178] [Bugs/问题反馈]
make[1]: *** [package/Makefile:110: /home/audy/openwrt/staging_dir/target-aarch64_cortex-a53_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/home/audy/openwrt'
make: *** [/home/audy/openwrt/include/toplevel.mk:230：world] 错误 2

**Comment by drwzheng** [comment]
me too

**Reply by lonelymylife** [reply]
stable yes;master no
